src/_guides/language/language-tour.md

```diff
@@ -3914,6 +3914,16 @@ main() {
 }
 {% endprettify %}
 
+<aside class="alert alert-info" markdown="1">
+**Version note:**
+A new form of function types was introduced in Dart 1.24. Unless you define
+fields that are function types, or you're doing complex things with
+generic methods, like passing them around, or defining an argument
+that's a function type that's also generic, you may not care about the
+new function-type syntax. For more information, see
+[Generic Function Type
+Alias](https://gist.github.com/eernstg/ffc7bd281974e9018f10f0cb6cfee4aa).
```

Comment:
When you go there, it points you to https://github.com/dart-lang/sdk/blob/master/docs/language/informal/generic-function-type-alias.md.